docs/DeveloperGuide.md

```diff
@@ -1,34 +1,503 @@
+
+### Delete a calendar item feature
+This feature is facilitated by `DeleteCommand`.
+The following sequence diagram show how the `execute()` operation works:<br/>
+<img src="https://raw.githubusercontent.com/AY2021S1-CS2113T-T12-2/tp/master/images/deleteCommand_SD.JPG" alt="" width="750"/><br/>
+
+Note: The command first extracts the task/event number from the user input prior to `convertTaskNumberToCalendarNumber` and `convertEventNumberToCalendarNumber` respectively. 
+
+
+### Find a calendar item feature
+This feature is facilitated by `FindCommand`.
+The following sequence diagram show how the `execute()` operation works when the user searches the entire calendar.:<br/>
+<img src="https://raw.githubusercontent.com/AY2021S1-CS2113T-T12-2/tp/master/images/findCommand_SD.JPG" alt="" width="500"/><br/>
+
+The search for tasks or events feature has a similar sequence diagram with a slight difference to the varying condition. Depending
+on whether the user searches for tasks or events, the condition will check for the instance of either the task or event respectively.
+
+### Print tasks feature
+This feature is facilitated by `PrintTasksCommand`.
+The following sequence diagram shows how the `execute()` operation works when the user wants to print the list of tasks stored in the program.
+![print_tasks_command_sd](images/PrintTasksCommand_SD.png)
+
+### Print events feature
+This feature is facilitated by `PrintEventsCommand`.
+The following sequence diagram shows how the `execute()` operation works when the user wants to print the list of events stored in the program.
+![print_events_command_sd](images/PrintEventsCommand_SD.png)
+
+### Print personal calendar feature
+This feature is facilitated by `PrintTimelineCommand`.
+The following sequence diagram show how the `execute()` operation works when the user wants to print the personalised calendar.:<br/>
+<img src="https://raw.githubusercontent.com/AY2021S1-CS2113T-T12-2/tp/master/images/PrintTimelineCommand_SD.png" alt="" width="750"/><br>
+
+### Prioritize a task feature 
+This feature is facilitated by `PrioritizeCommand`. 
+The following sequence diagram shows how the `execute()` operation works when the user wants to prioritize a task:<br/>
+<img src="https://raw.githubusercontent.com/AY2021S1-CS2113T-T12-2/tp/master/images/PrioritizeCommand_SD.png" alt="" width="750"/><br/>
+
+Note: It first extracts the task number from the user input prior to convertTaskNumberToCalendarNumber. 
+There will be a check in the function `markTaskAsImportant(calendarNumber)` to ensure that the calendar item being marked as important is a task.
+
+### Print prioritized tasks feature
+This feature is facilitated by `PrintPriorityCommand`.
+The following sequence diagram shows how the `execute()` operation works when the user wants to print all the prioritized tasks:<br/>
+<img src="https://raw.githubusercontent.com/AY2021S1-CS2113T-T12-2/tp/master/images/PrintPriorityCommand_SD.png" alt="" width="750"/><br/>
```

Comment:
![image](https://user-images.githubusercontent.com/18654251/97660644-2eb1ca00-1aad-11eb-93d3-dabca535d023.png)

Since the else field seems empty (unless there is a self-invocation, perhaps it would be better to use alt instead?
Also, it seems that the printImportantTask() arrow is not properly formatted